src/charm.py

```diff
@@ -181,6 +181,94 @@ def _on_synapse_nginx_pebble_ready(self, _: ops.HookEvent) -> None:
         self.pebble_service.replan_nginx(container)
         self._set_unit_status()
 
+    def _get_peer_relation(self) -> typing.Optional[ops.Relation]:
+        """Get peer relation.
+
+        Returns:
+            Relation or not if is not found.
+        """
+        peer_relation = self.model.get_relation(PEER_RELATION_NAME)
+        return peer_relation
```

Comment:
Do we usually do this? I'd rather `return self.model.get_relation(PEER_RELATION_NAME)` directly, though this is perfectly fine as well.

Reply:
Agree, changed!